### PR TITLE
Feature/get stream

### PR DESCRIPTION
**Pull request by MaksimK193**

### CODE REVIEW

File: SwiftUIStudy/Common/Managers/GetStreamManager.swift [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        chatClient = ChatClient(config: config)
+        streamChat = StreamChat(chatClient: chatClient)
+        connectUser()
+    }
+    
+    private func connectUser() {
+        let token = try! Token(rawValue: token)
+        chatClient.connectUser(userInfo: .init(id: "testuser2"),
+                               token: token) { error in
+            if let error = error {
```
Comment (IslamRustam) [inline SwiftUIStudy/Common/Managers/GetStreamManager.swift:32]: styling a bit off, no?

File: SwiftUIStudy/Flows/GetStreamChat/GetStreamChatView.swift [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import StreamChatSwiftUI
+
+struct GetStreamChatView: View {
+    let getStreamManager: GetStreamManager?
+    @StateObject private var viewModel: ChatChannelListViewModel
+    
+    public init(channelListController: ChatChannelListController? = nil,
+                getStreamManager: GetStreamManager) {
+        let channelListVM = ViewModelsFactory.makeChannelListViewModel(channelListController: channelListController,
+                                                                       selectedChannelId: nil)
```
Comment (IslamRustam) [inline SwiftUIStudy/Flows/GetStreamChat/GetStreamChatView.swift:19]: styling?

File: SwiftUIStudy/Common/Managers/GetStreamManager.swift (hunk 1 above)
Comment (MaksimK193) [reply SwiftUIStudy/Common/Managers/GetStreamManager.swift:32]: fixed

File: SwiftUIStudy/Flows/GetStreamChat/GetStreamChatView.swift (hunk 2 above)
Comment (MaksimK193) [reply SwiftUIStudy/Flows/GetStreamChat/GetStreamChatView.swift:19]: fixed

**Pull request merged**